src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -991,60 +1008,36 @@ namespace webnn_native { namespace ie {
         return {};
     }
 
-    MaybeError Graph::AddResample(const op::Resample* resample) {
-        auto input = mGraphNodeMap[resample->Inputs()[0].Get()];
+    MaybeError Graph::AddResample2d(const op::Resample2d* resample2d) {
+        auto input = mGraphNodeMap[resample2d->Inputs()[0].Get()];
         dimensions_t inputShape;
         ngraph_get_shape(input, &inputShape);
+        // WebNN axes.
+        auto axes = resample2d->GetAxes();
+        // sizes.
+        auto output = resample2d->Outputs()[0].Get();
+        auto outputShape = output->Shape();
```

Comment:
Merge L.1019 and L.1018 to one line or add a method in Resample2d like "GetOutputShape"
`auto outputShape = resample2d->Outputs()[0]->Shape().`